lib/vagrant-service-manager/binary_handlers/adb_binary_handler.rb

```diff
@@ -4,52 +4,6 @@ class ADBBinaryHandler < BinaryHandler
       def initialize(machine, env, options)
         super(machine, env, options)
       end
-
```

Comment:
@hferentschik Ya but I believe most of the higher level operations like download archive, extract it, binary path etc are almost same for all handlers except `build_download_url` which may be different for every handler.